src/Marksimple.php

```diff
@@ -1,14 +1,22 @@
-<?php # -*- coding: utf-8 -*-
```

Comment:
This `coding` comment should be removed because this is only for `Python` encoding definition.

Reply:
Not really. We discussed this also in this context and several IDEs and editors read them so that we use in our codex - https://github.com/inpsyde/php-coding-standards/

Reply:
Right. @bueltge, this the sanother PHP coding standard about defining the file `encoding`.

Thank you for your explanation.